compiler/rustc_codegen_llvm/src/common.rs

```diff
@@ -224,6 +224,10 @@ impl<'ll, 'tcx> ConstMethods<'tcx> for CodegenCx<'ll, 'tcx> {
         struct_in_context(self.llcx, elts, packed)
     }
 
+    fn const_vector(&self, elts: &[&'ll Value]) -> &'ll Value {
```

Comment:
Something very much like this [already exists](https://github.com/rust-lang/rust/blob/84ac80f1921afc243d71fd0caaa4f2838c294102/compiler/rustc_codegen_llvm/src/common.rs#L103-L106), does this need to be duplicated?